examples/time_frequency/plot_time_frequency_global_field_power.py

```diff
@@ -0,0 +1,142 @@
+intervals are computed as described in [3]_ across channels.
+
+The advantage of this method over summarizing the Space x Time x Frequency
+output of a Morlet Wavelet in frequency bands is relative speed and, more
+importantly, the clear-cut comparability of the spectral decomposition (the
+same type of filter is used across all bands).
+
+References
+----------
+.. [1]_ Hari R. and Salmelin R. Human cortical oscillations: a neuromagnetic
+        view through the skull (1997). Trends in Neuroscience 20 (1),
+        pp. 44-49.
+.. [2]_ Engemann D. and Gramfort A. (2015) Automated model selection in
+        covariance estimation and spatial whitening of MEG and EEG signals,
+        vol. 108, 328-342, NeuroImage.
+.. [3]_ Efron B. and Hastie T. Computer Age Statistical Inference (2016).
+        Cambrdige University Press, Chapter 11.2.
+"""
+# Authors: Denis A. Engemann <denis.engemann@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import somato
+
+###############################################################################
+# Set parameters
+data_path = somato.data_path()
+raw_fname = data_path + '/MEG/somato/sef_raw_sss.fif'
+
+# let's explore some frequency bands
+iter_freqs = [
+    ('Theta', 4, 7),
+    ('Alpha', 8, 12),
+    ('Beta', 13, 25),
+    ('Gamma', 30, 45)
+]
+
+###############################################################################
+# We create average power time courses for each frequency band
+
+# set epoching parameters
+event_id, tmin, tmax = 1, -1., 3.
+baseline = None
+
+# get the header to extract events
+raw = mne.io.read_raw_fif(raw_fname, preload=False)
+events = mne.find_events(raw, stim_channel='STI 014')
+
+frequency_map = list()
+
+for band, fmin, fmax in iter_freqs:
+    # (re)load the data to save memory
+    raw = mne.io.read_raw_fif(raw_fname, preload=True)
+    raw.pick_types(meg='grad', eog=True)  # we just look at gradiometers
+
+    # bandpass filter and compute Hilbert
+    raw.filter(fmin, fmax, n_jobs=1,  # use more jobs to speed up.
+               l_trans_bandwidth=1,  # make sure filter params are the same
+               h_trans_bandwidth=1)  # in each band and skip "auto" option.
+    raw.apply_hilbert(n_jobs=1, envelope=False)
+
+    epochs = mne.Epochs(raw, events, event_id, tmin, tmax, baseline=baseline,
+                        reject=dict(grad=4000e-13, eog=350e-6), preload=True)
+    # remove evoked response and get analytic signal (envelope)
+    epochs.subtract_evoked()  # for this we need to construct new epochs.
+    epochs = mne.EpochsArray(
+        data=np.abs(epochs.get_data()), info=epochs.info, tmin=epochs.tmin)
+    # now average and move on
+    frequency_map.append(((band, fmin, fmax), epochs.average()))
+
+###############################################################################
+# Now we can compute the Global Field Power
+
```

Comment:
I would make it a full block of text and don't have the inline comments that won't pop out